### PR TITLE
updated syntax

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Summary

Resolves issue #16662 
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/convolvernode/normalize/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ ajaxRequest.open('GET', 'concert-crowd.ogg', true);
 ajaxRequest.responseType = 'arraybuffer';
 
 ajaxRequest.onload = function() {
-  var audioData = ajaxRequest.response;
```
Comment (wbamberg) [inline files/en-us/web/api/convolvernode/normalize/index.md:48]: Line 41 in this file, which I can't select for comment, can be:
`let soundSource, concertHallBuffer;`

....although I would prefer to write:

```
let soundSource;
let concertHallBuffer;
```

File: files/en-us/web/api/cssstyledeclaration/length/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ The following gets the number of explicitly set styles on the following HTML ele
 JavaScript code:
 
 ```js
-var myDiv = document.getElementById('div1'); var divStyle = myDiv.style; var len = divStyle.length; // 6
+const myDiv = document.getElementById('div1'); const divStyle = myDiv.style; const len = divStyle.length; // 6
```
Comment (wbamberg) [inline files/en-us/web/api/cssstyledeclaration/length/index.md:33]: not strictly in scope but can we have separate lines here:

```suggestion
const myDiv = document.getElementById('div1');
const divStyle = myDiv.style;
const len = divStyle.length; // 6
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thank you @gullapalli9482 !

File: files/en-us/web/api/convolvernode/normalize/index.md (hunk 1 above)
Comment (gullapalli9482) [reply files/en-us/web/api/convolvernode/normalize/index.md:48]: My bad, this is the disadvantage of editing files in the terminal, ubuntu terminal has a bad background color. I'll make sure to check the files multiple times before committing them.

File: files/en-us/web/api/cssstylesheet/insertrule/index.md [hunk 3]
```diff
@@ -115,8 +115,8 @@ function addStylesheetRules (rules) {
       j = 0;
     }
 
-    for (var pl = rule.length; j < pl; j++) {
-      var prop = rule[j];
+    for (let pl = rule.length; j < pl; j++) {
+      let prop = rule[j];
```
Comment (wbamberg) [inline files/en-us/web/api/cssstylesheet/insertrule/index.md:119]:
```suggestion
      const prop = rule[j];
```

Sorry I missed this one!

**Comment by gullapalli9482** [comment]
Are there any other files to be modified?

File: files/en-us/web/api/convolvernode/normalize/index.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ ajaxRequest.open('GET', 'concert-crowd.ogg', true);
...
+  const audioData = ajaxRequest.response;
```
Comment (wbamberg) [inline files/en-us/web/api/convolvernode/normalize/index.md:48]: Line 41 in this file should also use `let`.

**Review by wbamberg — APPROVED** [review]
👍 looks great, @gullapalli9482 , thank you!

**Comment by wbamberg** [comment]
By the way, you don't (IMO) have to merge main back into your branch unless you have conflicting files (and GitHub will tell you if you do have conflicting files).

**Pull request merged**